pkg/cmd/project/create/create.go

```diff
@@ -0,0 +1,140 @@
+package create
+
+import (
+	"fmt"
+
+	"github.com/cli/cli/v2/pkg/cmdutil"
+
+	"github.com/cli/cli/v2/pkg/cmd/project/shared/format"
+	"github.com/cli/cli/v2/pkg/cmd/project/shared/queries"
+
+	"github.com/cli/cli/v2/internal/tableprinter"
+	"github.com/cli/go-gh/v2/pkg/api"
+	"github.com/shurcooL/githubv4"
+	"github.com/spf13/cobra"
+)
+
+type createOpts struct {
+	title     string
+	userOwner string
+	orgOwner  string
+	ownerID   string
+	format    string
+}
+
+type createConfig struct {
+	tp     *tableprinter.TablePrinter
+	client *api.GraphQLClient
+	opts   createOpts
+}
+
+type createProjectMutation struct {
+	CreateProjectV2 struct {
+		ProjectV2 queries.Project `graphql:"projectV2"`
+	} `graphql:"createProjectV2(input:$input)"`
+}
+
+func NewCmdCreate(f *cmdutil.Factory, runF func(config createConfig) error) *cobra.Command {
+	opts := createOpts{}
+	createCmd := &cobra.Command{
+		Short: "Create a project",
+		Use:   "create",
+		Example: `
+# create a new project owned by user monalisa with title "a new project"
+gh project create --user monalisa --title "a new project"
+
+# create a new project owned by org github with title "a new project"
+gh project create --org github --title "a new project"
+
+# create a new project owned by the current user with title "a new project"
+gh project create --user '@me' --title "a new project"
+
+# add --format=json to output in JSON format
+`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := queries.NewClient()
```

Comment:
I think I had looked into doing that, and it ended up being a pretty significant change because I'm using the github.com/cli/go-gh/v2/pkg/api `GraphQLClient` and it's `Query` method https://github.com/cli/go-gh/blob/v1.2.1/pkg/api/client.go#L138 

which isn't the same as what's available in cmdutil.Factory